frontend/testing/lib/actions.test.ts

```
@@ -303,827 +133,90 @@ describe("actions", () => {
-      it("successfully creates a playlist", async () => {
-        const mockResponse = {
-          ok: true,
-          json: () => Promise.resolve({ data: mockPlaylistData }),
+    describe("Bug Report Actions", () => {
+      it("should create bug report", async () => {
+        const formData = {
+          title: "Test Bug",
+          description: "Test Description",
+          email: "email@example.com",
+          path: "/home",
+          type: "bug" as "bug",
```

ESLint

[error] Expected a `const` instead of a literal type assertion.

(@typescript-eslint/prefer-as-const)